prosody/Dockerfile

```diff
@@ -38,6 +38,10 @@ RUN \
     && apt-cleanup \
     && rm -rf /tmp/usr /var/cache/apt
 
+RUN \
+    curl -4so /prosody-plugins/mod_turncredentials.lua \
```

Comment:
you need to add : apt-get install curl

Reply:
fixed in 5753213

Reply:
You could also just 
ADD ADD https://hg.prosody.im/prosody-modules/raw-file/tip/mod_turncredentials/mod_turncredentials.lua /prosody_plugins
And avoid adding curl in base image